py-polars/polars/expr/expr.py

```diff
@@ -240,6 +240,21 @@ def function(s: Series) -> Series:  # pragma: no cover
 
         return self.map(function)
 
+    @classmethod
+    def read_json_string(cls, value: str) -> Self:
```

Comment:
This would be the only classmethod for `Expr`. I think it would be more consistent with the rest of the API to offer this as as a function, e.g. `pl.read_json_string` or something similar.

Reply:
> This would be the only classmethod for `Expr`. I think it would be more consistent with the rest of the API to offer this as as a function, e.g. `pl.read_json_string` or something similar.

It is very unclear what the return type is in that case. I thought about this, but I don't think that will be pretty.